app/src/main/java/com/example/keybindhelper/dto/Group.java

```diff
@@ -53,8 +53,6 @@ public Group(){}
 
     /**
      * Initializes the group with a name and projectID
-     * @param name
-     * @param projectID
```

Comment:
You can see the parameter names in source code.  These should be described in KDoc.  Listing them, without a description, does not add value or reduce technical debt.